scripts/abins/input/tester.py

```diff
@@ -205,10 +199,6 @@ def save_ab_initio_test_data(cls, ab_initio_reader, seedname):
         """
 
         data = cls._get_reader_data(ab_initio_reader)
-        # Unpack advanced parameters into a dictionary for ease of comparison later.
-        # It might be wise to remove this and store escaped strings for consistency,
-        # but for now we don't want to disturb the old test data so follow its format.
-        data["attributes"]["advanced_parameters"] = json.loads(data["attributes"]["advanced_parameters"])
```

Comment:
Is the intention of removing this line to stop the advanced_parameters being written out to file? I saw in the description on the PR that was one of the changes that had been made.
It looks like this line just formatted the json string into a dictionary so the parameters will still be written out a few lines later on even without this line

Reply:
Good catch! On further investigation, the block is introduced as an obscure side-effect of the caching system. This kind of thing is exactly why I intend to overhaul that system; it _does_ make sense to include the advanced parameter state in the cache of an overall Abins run, but not to associate it with an input file.

For now we can explicitly drop it from the saved data with `del`. Hopefully when the caching is reworked it can be separated entirely from this.